src/css/index.scss

```diff
@@ -0,0 +1,103 @@
+.Info-text1{ 
+    width: 660px;
+    height: 630px;
+    background-image: repeating-linear-gradient(120deg,#3498bd,#8e44ad); 
+    color: white;
+    font-family: sans-serif;
+   }
+   
+   .h2-qu1{
+      padding: 40px 40px;
+      font-size: 40px;
+   }
+   .h2-qu2{
+      padding: 40px 40px;
+      font-size: 40px;
+      color: black;
+      font-family: sans-serif;
+      font-weight: bold;
+   }
+   
+   tr{
+       font-size: 16px;
+       line-height: 40px;   
+   }
+   .bold-font{
+       font-weight: bold;
+       margin-top: 50px;
+       padding: 3px 40px 0;
+       text-align: left;
+       line-height: normal;
+       position: relative;  
+   }
+   a{
+       color: white;
+   }
+   .Info-text2{ 
+    width: 660px;
```

Comment:
Keep the css content have same indentation

Reply:
if you npm build environment is working, you shouldn't be able to submit this pull request.
We really need to sort out your coding procedures. Otherwise own review is not benefit for you at all.